client/src/App.tsx

```diff
@@ -21,6 +23,9 @@ function App() {
     currency: '',
   });
   const [fxRates, setFxRates] = useState<FxRates>({CAD: 1, EUR: 1, USD: 1});
+  const [isSnackbarOpen, setIsSnackbarOpen] = useState(false);
+  const [snackbarMessage, setSnackbarMessage] = useState('');
+  const [alertColor, setAlertColor] = useState();
```

Comment:
For useState here we should give it a type and an initial value. See the other comment for how you get AlertColor.

`useState<AlertColor|null>(null);`